docs/src/how-tos/contribute_to_or.md

```diff
@@ -0,0 +1,27 @@
+---
+id: howto-contribute-or
+title: How-To Contribute to OpenRefine's documentation
+sidebar_label: How-To Contribute to docs
+---
+
+
+If you are interested to help make OpenRefine’s documentation better, here's some informations.
+
+The documentation is separated in the Product Reference part, and How-to. There is some documentation mainly written for Users, and there is some documentation more geared toward the development of OpenRefine. 
+
+First, you have to download the OpenRefine project on your computer. To do so, you can use `git` or GitDesktop. You can also use a git client integrated in an IDE like Eclipse.
+
+Then, you can configure [Docusaurus](https://docusaurus.io/), the tool the OpenRefine uses to manage it's documentation library.
```

Comment:
```suggestion
Then, you can configure [Docusaurus](https://docusaurus.io/), the tool which OpenRefine uses to manage its documentation library.
```

Reply:
@kushthedude: thank you very much for your corrections!

I'm even before the draft stage with this document, I will for sure need more help.

Regards, A.